### ISSUE TITLE
Parse ClassPath

### ISSUE THREAD

**Issue by KitsuneDev** [javalite/jar-explorer#2]
Hey!
Just a quick suggestion: it would be really nice if we could input a list of jars separated by ; instead of a root directory.

**Comment by ipolevoy** [comment]
@GabrielTK maybe, but generally it was designed to find things in the maven repo. You can copy jars to a directory and point to it.

**Comment by KitsuneDev** [comment]
Yeah, but it would be easier to copy a class path and use it. Going to send a PR tomorrow.

On Mar 2, 2017 3:19 AM, "Igor Polevoy" <notifications@github.com<mailto:notifications@github.com>> wrote:

@GabrielTK<https://github.com/GabrielTK> maybe, but generally it was designed to find things in the maven repo. You can copy jars to a directory and point to it.

**Comment by ipolevoy** [comment]
@GabrielTK sure, as long as the original method works as well

**Comment by damjad** [comment]
@ipolevoy This issue is resolved. Please accept the pull request https://github.com/javalite/jar-explorer/pull/4.
